app/data/DataStores.scala

```diff
@@ -2,29 +2,20 @@ package data
 
 import com.gu.atom.data._
 import com.gu.atom.publish._
-import com.gu.contentatom.thrift.{Atom, AtomData}
-import com.gu.contentatom.thrift.atom.media.MediaAtom
+import com.gu.contentatom.thrift.Atom
 import com.gu.media.pluto.PlutoProjectDataStore
 import com.gu.media.upload.UploadsDataStore
 import com.gu.media.{CapiPreviewAccess, PlutoDataStore}
-import com.gu.scanamo.DynamoFormat
 import util.atom.MediaAtomImplicits
```

Comment:
It is unnecessary, but I don't think its related to this PR, I can remove it in another PR.